### ISSUE TITLE
The following configuration fields are not understood: {'database'}

### ISSUE THREAD

**Issue by theodotos** [matrix-org/sygnal#109]
I am running sygnal following the instructions in the README: `python -m sygnal.sygnal`

I am using sqlite3 exactly like in the sample configuration:

```
database:
  name: sqlite3
  args:
    dbfile: sygnal.db
```

But I get this:

```
The following configuration fields are not understood: {'database'}
```

Any idea?

**Comment by clokep** [comment]
@theodotos What version of Sygnal are you running?

**Comment by theodotos** [comment]
@clokep 

I cloned it recently from git:

```
# git log | head
commit 8567224dd5112599cbe0a997aaccc075d7b5c254
Merge: c8a1216 1f33ce1
Author: Will Hunt <will@half-shot.uk>
Date:   Mon Apr 6 21:41:49 2020 +0100

    Merge pull request #97 from csett86/fix-sample-config
    
    Fix the given sample config to match the implementation

commit 1f33ce1f3208602b173db28eba0142c4281d828e
```

**Comment by theodotos** [comment]
@clokep 

I pulled the latest version:

```
# git log | head
commit 44aa835086af0567061d26a47dd10a6aec912ba9
Author: ismailgulek <gulekismail@gmail.com>
Date:   Fri Apr 24 16:56:50 2020 +0300

    Update changelog
    
    Signed-off-by: ismailgulek <gulekismail@gmail.com>

commit dc3ecc3172a4937eb2ac586c20b40a7e38950b18
Author: ismailgulek <gulekismail@gmail.com>
```

Same:

```
# python -m sygnal.sygnal
The following configuration fields are not understood: {'database'}

```

**Comment by babolivier** [comment]
Hey there, and thanks for the report! I've opened https://github.com/matrix-org/sygnal/pull/111 to fix this issue.

**Comment by babolivier** [comment]
Turns out #100 is already fixing this.

**Comment by theodotos** [comment]
I didn't mention that the service runs, when this happens but does not listen to port 5000.

**Comment by babolivier** [comment]
For what it's worth, this log line is only a warning and shouldn't impact Sygnal's ability to start and run properly (it's much more of a logging mess than an actual bug). I think your port 5000 issue might be unrelated.

**Comment by anilk9096** [comment]
hi ,

when can we expect this fix , with out sygnal server we cant migrate our app due to push notifications .

any alternate for sygnal . kindly suggest

**Comment by richvdh** [comment]
fixed by #100 I believe
